src/args.rs

```diff
@@ -33,7 +36,11 @@ impl Args {
             } else if let Some(str) = cli.restic_password {
                 Password::Plain(str)
             } else {
-                unreachable!("Error in Config: neither password_command nor password_file found. Please open an issue if you see this.")
+                let pwd = Self::read_password_from_stdin();
+                // There's no way to hand over the password to restic directly, so we use the env
+                // variable instead.
+                env::set_var("RESTIC_PASSWORD", &pwd);
```

Comment:
Not sure if this here's the right place to set the env variable. I guess it would be better to do it in `restic.rs` instead. Because I reused `Password::Plain`, it would then also trigger if the env variable is already set. There's no real harm in doing it twice.

What do you think?

Reply:
I think the best way is to to use [std::Command::env](https://doc.rust-lang.org/std/process/struct.Command.html#method.env) and set the password only for the restic subprocesses in `restic.rs`.